lib/src/kuzzle/request.dart

```diff
@@ -11,7 +11,7 @@ class KuzzleRequest {
       this.index,
       this.jwt,
       this.requestId,
-      this.refresh,
+      this.waitForRefresh = false,
```

Comment:
This should not be `false` by default but some kind of undefined value because for example the default value for `wait_for_refresh` is `wait_for` for the security controller

Reply:
Yes but it's the security controller that set `waitForRefresh` to true and if it is true then the query method will set refresh to `wait_for`

Reply:
IMHO the security controller in the SDK should not set default value but Kuzzle should

Reply:
But in the sdk if `waitForRefresh` is false then it does not set any value for refresh, Kuzzle will receive a null value

Reply:
Ok I see but so the problem is that if you want to pass `refresh: 'false'` for one of the security controller method then it will not work and the default value (`wait_for`) will be used